feature-libs/organization/administration/components/unit/links/approvers/unit-approver-list.service.ts

```diff
@@ -5,11 +5,15 @@ import {
   EntitiesModel,
   PaginationModel,
 } from '@spartacus/core';
-import { OrgUnitService } from '@spartacus/organization/administration/core';
+import {
+  OrgUnitService,
+  B2BUserService,
+} from '@spartacus/organization/administration/core';
```

Comment:
will it create a circular dependency?

Reply:
we are in @spartacus/organization/administration/components